core/src/main/scala/com/rallyhealth/vapors/core/interpreter/ExprInput.scala

```diff
@@ -8,7 +8,7 @@ final case class ExprInput[V](
   factTable: FactTable,
 ) {
 
-  // TODO: Remove
+  @deprecated("Use withValue instead.", "0.10.0")
```

Comment:
Do you want to include a target removal version?  I think that we should start doing this.

Reply:
I don't really know. I might remove it before 1.0, or I might remove it before that.